EngineLayer/ClassicSearch/ClassicSearchEngine.cs

```diff
@@ -203,5 +249,7 @@ private int GetFirstScanWithMassOverOrEqual(double minimum)
             // index of the first element that is larger than value
             return index;
         }
+
```

Comment:
remove extra empty lines between brackets, i've noticed it a couple places elsewhere too